dense_tiles/dense_tiles.cpp

```diff
@@ -139,6 +159,8 @@ int main(int argc, char* argv[]) {
     for (unsigned int i = 0; i < grid.size(); ++i) {
         if (grid[i]) {
             sorter.emplace_back(grid[i], i);
+        } else {
+            sorter.emplace_back(0, i);
```

Comment:
If the else clause missed, tiles without any nodes would not be printed. I had a use case in my master thesis where it is useful to have a list of all tiles—even the empty ones.

I will add a comment in the source code which explains it.

I will set the default value of `min_nodes` to 1 in order to keep the current behaviour and be backward compatible.